doc/qbittorrent.1.md

```diff
@@ -3,34 +3,165 @@
+
+# ENVIRONMENT
 
-**`--no-splash`** Disables splash screen on startup.
+_OPTIONS_ may be supplied via environment variables with equivalent names. 
 
-**`--webui-port=x`** Changes Web UI port to x (default: 8080).
+For an option named **parameter-name**, the corresponding environment variable name is **QBT_PARAMETER_NAME**. In other words, the parameter name is converted to upper-case, any hyphens (**-**) are replaced by underscores (**\_**), and **QBT_** is prepended to it.
 
+To pass flag values, set the variable to **1** or **TRUE**. For example, to disable the splash screen: **QBT_NO_SPLASH=1**
+
+# FILES
+
+qBittorrent complies with the Freedesktop XDG Base Directory specification. TODO: not always, see "data" folder.
+Typically, **\$XDG_CONFIG_HOME** is _~/.config_ and **$XDG_DATA_HOME** is _~/.local/share_.
+
+TODO: did I miss any?
+
+_**$XDG_CONFIG_HOME**/qBittorrent/qBittorrent.conf_
+
+:   Main configuration file.
+
+_**$XDG_CONFIG_HOME**/qBittorrent/qBittorrent-data.conf_
+
+:   This is where all-time statistics are saved.
+
+_**$XDG_CONFIG_HOME**/qBittorrent/rss/feeds.json_
+
+:    RSS feeds configuration.
+
+_**$XDG_DATA_HOME**/data/qBittorrent/BT_backup/_
+
+:    This directory contains the _.fastresume_ files, which is the mechanism qBittorrent uses to keep track of the state of each added torrent across restarts. In addition, a copy of each of the added torrents' meta-files is kept here, with the torrents' hashes as the file names.
+
+_**$XDG_DATA_HOME**/data/qBittorrent/GeoIP/GeoLite2-Country.mmdb_
+
+:    Database for IP <-> country flag resolution in the "Peers" tab.
+
+_**$XDG_DATA_HOME**/data/qBittorrent/logs_
+
+:    Execution logs.
+
+_**$XDG_DATA_HOME**/data/qBittorrent/nova3/_
+
+:    This directory contains the Python files for the Search Engine functionality.
+
+_**$XDG_DATA_HOME**/data/qBittorrent/nova3/engines/_
+
+:    Directory for custom search engine code and files.
+
+_**$XDG_DATA_HOME**/data/qBittorrent/rss/articles_
+
+:    RSS feed storage.
+
+TODO: document lock files and their interaction with the configuration option.
+
+/tmp/qtsingleapp-qBitto-1809-3e8-lockfile <- this name will vary
+_**$XDG_CONFIG_HOME**/qBittorrent/rss/storage.lock
+_**$XDG_DATA_HOME**/data/qBittorrent/BT_backup/session.lock
+_**$XDG_DATA_HOME**/data/qBittorrent/rss/articles/storage.lock
+
+# NOTES
+
+Refer to the wiki for addtional information, including guides on how to compile qBittorrent from source: https://github.com/qbittorrent/qBittorrent/wiki
 
 # BUGS
-If you find a bug, please report it at http://bugs.qbittorrent.org
+
+If you find a bug, please report it at https://bugs.qbittorrent.org
+
+# EXAMPLE
+
+TODO
+
+# SEE ALSO
+
+**btcheck(1)**, **mktorrent(1)**
```

Comment:
Those software doesn't belong to this project. IMO no point mentioning them in our man page and man page isn't a wiki...

Reply:
Yeah but isn't the point of this section to mention related software that's potentially useful, even though it might not belong to the same project? It serves helps discoverability of related programs.

Reply:
Probably just me, I don't feel like promoting other programs and be held liable when they go wrong...

Reply:
I suggested these ones because they are useful in general and very mature. But if you really, insist, I don't mind not including this.